compute_tools/src/compute.rs

```diff
@@ -400,4 +402,39 @@ impl ComputeNode {
 
         Ok(())
     }
+
+    /// Select `pg_stat_statements` data and return it as a stringified JSON
+    pub async fn collect_insights(&self) -> String {
+        let mut result_rows: Vec<String> = Vec::new();
+        let connect_result = tokio_postgres::connect(self.connstr.as_str(), NoTls).await;
+        let (client, connection) = connect_result.unwrap();
+        tokio::spawn(async move {
+            if let Err(e) = connection.await {
+                eprintln!("connection error: {}", e);
+            }
+        });
+        for message in (client
+            .simple_query(
+                "
+SELECT
+    row_to_json(pg_stat_statements)
+FROM
+    pg_stat_statements
+WHERE
+    userid != 'cloud_admin'::regrole::oid
+ORDER BY
+    (mean_exec_time + mean_plan_time) DESC
+LIMIT 100",
+            )
+            .await)
+            .unwrap()
```

Comment:
Potentially, it can panic here. Make it safer a bit.

I assume that here it is less dangerous, but still.

```rust
row.get(0).unwrap().to_string()
```

Reply:
> I assume that here it is less dangerous

Yes, if it panics, then request handler just panics and nothing special will happen (I suppose)

Is your code snippet a suggestion? Then I'm not sure where to put it?

Reply:
nope, there are two places with `unwrap`

Reply:
I got rid of unnecessary unwraps